Makefile

```diff
@@ -51,7 +52,7 @@ $(OUT)/cpu_instrs.h: tests/cpu_instrs.gb tests/rom2h.c
 
 $(OUT)/bench: $(OUT)/cpu_instrs.h prof.h bench.c gameboy.h
 	$(VECHO) "  CC+LD\t$@\n"
-	$(Q)$(CC) -o $@ bench.c
+	$(Q)$(CC) -DENABLE_LCD=0 -o $@ bench.c cpu.c
```

Comment:
Change the sequence from `bench.c cpu.c` to `cpu.c bench.c`, which aids the compiler to understand the expected dependency.